NYPLCardCreator/Helpers/NYPLColor.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NYPLColor.swift
+//  NYPLCardCreator
+//
+//  Created by Ernest Fan on 2021-09-24.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import UIKit
+
+private enum ColorAsset: String {
+  case primaryBackground
+  case primaryText
+  case action
+  case disabledFieldText
+}
+
+class NYPLColor {
+  static var primaryBackgroundColor: UIColor {
```

Comment:
I was wondering if it's possible to use for light mode the adaptable semantic colors that iOS 13 provides (e.g. `UIColor.link`, that was used in IntroductionViewController).... something like
```
if light mode and iOS 13 {
  return UIColor.systemBackground // or UIColor.label for `primaryTextColor`, etc
} else {
  // your implementation
} 
```

Reply:
Sure I can do that, I will do the same when I submit PR for Simplified-iOS